### PR TITLE
WAR-1340 : Links to open Testcase from Suite

### PR DESCRIPTION
**Pull request by priyaarun23**
Dynamically formed links of Testcase(from Suite) could be opened in Linux as well as Windows.

### CODE REVIEW

**Comment by priyaarun23** [comment]
@sanikakulkarni 
Request you to review the same.

**Comment by priyaarun23** [comment]
@sanikakulkarni Updated the fix according to the Design change.

**Old Design** :  Opening a case in suite opens up its corresponding XML in new browser tab.
**New Design** : Opening a case in suite opens up its corresponding Katana UI with the details in new browser tab.

Shown demo to the user and they agree for the fix.

Hence requesting you to review the same, as they are in need of this feature to ease their work.

**Note** : Fixed WAR-1350 in this ticket itself. _WAR-1350 : To open Suite from Project._

File: katana/static/app/controllers/projectCaptureCtrl.js [hunk 1]
```diff
@@ -36,6 +36,36 @@ app.controller('projectCapCtrl', ['$scope', '$http', '$routeParams', '$controlle
...
+          url = filepath.split('..')[1];      
+          splitDir = url.split('/Suites')[1]; 
+          finalUrl = "#/testsuite"+splitDir+"/none";
+          window.open(finalUrl);
+
+       }
+
+        else{
+          
+          splitPath = filepath.split("/").pop(-1);
```
Comment (kaheichan) [inline katana/static/app/controllers/projectCaptureCtrl.js]: Need some comments for this block of code.

**Comment by kaheichan** [comment]
verified functionality, pending review on code quality

**Comment by priyaarun23** [comment]
Added Comment Lines.

**Comment by kaheichan** [comment]
@priyaarun23 Saw the fix, it looks good
Just one last thing before approving, the error message for opening an non-existing tc is 500 undefined
Can we have a more meaningful error message for that? other than that the functionality of the PR is verified

**Comment by priyaarun23** [comment]
@kaheichan : '500 undefined error' for the **non-existing file** is throughout the Katana application. This  could be handled in WAR-1222. Hence requesting you to approve this ticket.

**Comment by kaheichan** [comment]
Hi @priyaarun23 
WAR-1449 is created in Jira for the error message

**Pull request merged**